recipes/pysr/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set version = "0.7.0a1" %}
+{% set pyjulia_version = "0.5.7" %}
+
+package:
+  name: pysr
+  version: {{ version }}
+
+source:
+  - url: https://github.com/MilesCranmer/PySR/archive/v{{ version }}.tar.gz
+    sha256: f002094c18182e6b00f6e0a13452bc7f80d891623932f372e8e6aa15fdf14ce4
+    folder: pysr
+  # PyJulia is not on conda-forge, so we also build it here.
+  - url: https://files.pythonhosted.org/packages/33/90/ba856cddadd923c94b7ce07dd3623d37cfea22b815076f0c70a47d867a3e/julia-{{ pyjulia_version }}.tar.gz
+    sha256: f24b9788f3e30f33d53fc8854f2fcc54193c84412149cfbc652bde2edd6e8a85
+    folder: pyjulia
+
+build:
+  # conda-forge julia is not available on windows:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install ./pyjulia -vv && {{ PYTHON }} -m pip install ./pysr -vv && 
+   {{ PYTHON }} -c 'import pysr; pysr.install()'"
+
+requirements:
+  build:
+    - python
+    - pip
+    - julia
+  host:
+    - python
```

```suggestion
    - python >=3.7
```
necessary for noarch